cost-analyzer/templates/query-service-deployment-template.yaml

```diff
@@ -47,7 +62,7 @@ spec:
           volumeMounts:
             - name: persistent-configs
               mountPath: /var/configs
-            - name: persistent-db
+            - name: storage
               mountPath: /var/db
           securityContext:
             runAsUser: 0
```

Comment:
is user 0 needed?

Reply:
@jessegoodier I actually think it is. This is an init container that sets volume perms before the query replica runs. User 0 will allow it to take volumes with arbitrary perms and ensure they are writable by the pod